### DISCUSSION TITLE
weird responsiveness bug on mobile

### DISCUSSION THREAD

**Discussion by prljav** [requarks/wiki#7494] [Error / Bug Report]
video below, not much more to add
https://github.com/user-attachments/assets/a1ce6903-e774-40c2-97d4-8d75395dd421

edit:
1080x2408 resolution

**Comment by tandibar** [comment]
I have something very similar:
https://github.com/user-attachments/assets/d126e776-97a4-422e-a3bf-16093a8ed04f
might be related or not :-)

This only happens on some pages, not on all.
